client/src/Pages/Home.js

```diff
@@ -1,67 +1,88 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import Row from '../components/Row';
 import requests from '../services/requests';
 
-function Home({ addToWatchList, isOnWatchList }) {
+function Home({ isFavorite, isOnWatchList, addToWatchList, addToFavorites }) {
   return (
-    <>
+    <WrapWrapper>
       <Row
```

Comment:
All your rows seem to be pretty much the same expect for the title and the fetch URL. Maybe you could write a loop that draws them automatically .

E.g. have an array with the information you need for each row:

`const rowData = [{title: 'NETFLIX ORIGINALS', url: 'fetchNetflixOriginals'}, {title: 'TRENDING', url: 'fetchTrending'},... `

`function makeMovieRows(rowData) {
let movieRows = []
 for(let i = 0; i <= rows.lenght; i++) {
movieRows.push('<Row title={rowData.**title**} isFavorite={isFavorite} isOnWatchList={isOnWatchList}
        addToWatchList={addToWatchList}
        addToFavorites={addToFavorites}
        fetchUrl={requests.**rowData.url**} />)
}
return movieRows
}
`

And then you simply need to call the function. 
Just an idea, not sure if it really is better than what you already have 😸